sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/math.scala

```diff
@@ -312,3 +315,90 @@ case class Logarithm(left: Expression, right: Expression)
     """
   }
 }
+
+case class Round(child: Expression, scale: Expression) extends Expression {
+
+  def this(child: Expression) = {
+    this(child, Literal(0))
+  }
+
+  def children: Seq[Expression] = Seq(child, scale)
+
+  def nullable: Boolean = true
```

Comment:
OK, let's say if the both children are literals e.g. `Literal(123.0, FloatType)` and `Literal(1. IntegerType)`, still be nullable?